### DISCUSSION TITLE
Express project maturity

### DISCUSSION THREAD

**Discussion by HarelM** [maplibre/maplibre#140] [Ideas]
In one of the threads of discussion #132 we started talking about experimental projects.
In the TSC today, it was noted that these projects are important to the future of maplibre in order to be able to incubate the future technologies and allow the community to gather around things that are still not mature.
Having said that, there's still a need, from my point of view, to let the users of a project understand fast what is the status of this project when using either the maplibre.org site, going into the maplibre org in github or going to the project repo.

I propose to make sure a repo is experimental by
1. Not making it public in the maplibre.org site, I think maplibre.org should only list repo/projects that are in use in production by X number of users/companies (I'm not sure how to verify or validate this, but I would compare it to a company's website where you have products that are "ready for sale" only).
2. Add a tag to the repo
3. Make it super clear in the readme of that repo

Any thoughts and insights to this would be great!
cc @maxammann 

P.S. this is orthogonal to the tiers discussion, as a project can be both experimental and still core, supported or hosted.

**Comment by lseelenbinder** [comment]
I agree in principle.

I would say I don't think a project can be core and experimental, but maybe I'm selling us short on that one. :smile: I'm fine with this as-is.

**Comment by maxammann** [comment]
> Not making it public in the maplibre.org site

I think any maplibre Github repo site should be allowed to push its content to the maplibre.org domain. Else we quickly get into trouble when e.g. the maplibre-rs project wants to publish its api docs somewhere. In that case we need a more sophisticated web hosting solution probably to deploy to a different domain.

As long as experimental/unstable projects are not advertised on the home page I think it is fine. Right now all experimental projects are clearly marked on the maplibre.org website.

I honestly think that nobody cares which projects are listed on the maplibre.org site vs the github.com/maplibre pages.

> Add a tag to the repo

I'm not sure which tags you mean here. Github tags are probably not reviewed by a lot of users.

> Make it super clear in the readme of that repo

We should do that for all the experimental repos! For maplibre-rs I did that. This is absolutely valuable information for users.

**Reply by maxammann** [reply]
I also think that the website could need a redesign! I only improved it regarding structure and the technology used -> hugo instead of plain HTML. So we definitely could invest some money/time there.

**Reply by HarelM** [reply]
I meant the home page, sorry if that wasn't clear. API docs should be available for every repo, experimental or not.
